### ISSUE TITLE
Discord notify failed because priority is not valid config key

### ISSUE THREAD

**Issue by SnowSakura225** [Darkness4/fc2-live-dl-go#21]
I want to setup discord notification and i have set noPriority to true in the config but still failed. maybe i do something wrong?
this is my log
```
2023-12-16T12:00:50Z INF watch/watch_command.go:126 > using shoutrrr 
2023-12-16T12:00:50Z ERR watch/watch_command.go:135 > notify failed | error=failed to send discord notification: priority is not a valid config key [avatar avatarurl color colordebug colorerror colorinfo colorwarn json splitlines title username]
```

and my notifier config
```
notifier:
  enabled: true
  includeTitleInMessage: false
  noPriority: true
  urls:
    - 'discord://token@webhooktoken'

  notificationFormats:
    panicked:
      enabled: true
	    title: "panicked"

    downloading:
      enabled: true
      title: '{{ .Labels.EnglishName }} is streaming'

    finished:
      enabled: true
      title: '{{ .Labels.EnglishName }} stream ended'

    error:
      enabled: true
      title: 'stream download of {{ .Labels.EnglishName }} failed'

    canceled:
      enabled: true
      title: 'stream download of {{ .Labels.EnglishName }} canceled'
```

**Comment by Darkness4** [comment]
Oh, sorry. That's embarassing, I forgot to release a new version. Will publish right now.

**Comment by Darkness4** [comment]
Closed by commit 413c995b38fc8602d6363e309b9fa6bf0ef69585, and released in version [v1.4.1](https://github.com/Darkness4/fc2-live-dl-go/releases/tag/v1.4.1).
